### PR TITLE
Support for ADMIN scope

### PR DESCRIPTION
**Pull request by tkrille**
ADMIN is a scope that allows full access to any resource.

### CODE REVIEW

**Comment by tkrille** [comment]
ITs coming soon.

File: src/main/webapp/WEB-INF/rest-security.xml [hunk 1]
```diff
@@ -77,6 +77,9 @@
                 <bean class="org.osiam.security.authorization.MeScopeAccessDecisionVoter">
                     <constructor-arg ref="scopeVoter"/>
                 </bean>
+                <bean class="org.osiam.security.authorization.AdminScopeAccessDecisionVoter">
```
Comment (dacrome) [inline src/main/webapp/WEB-INF/rest-security.xml]: the admin scope voter should be checked first

Comment (tkrille) [inline src/main/webapp/WEB-INF/rest-security.xml]: done

**Comment by tkrille** [comment]
ITs: osiam/connector4java-integration-tests#238

**Pull request merged**